cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt

```diff
@@ -135,23 +147,40 @@ private class CampaignManagerImpl(
     fun getGdprPmConfig(pmId: String?, pmTab: PMTab): Either<PmUrlConfig> = check {
         val uuid = dataStorage.getGdprConsentUuid()
         val siteId = dataStorage.getPropertyId().toString()
+        val childPmId: String? = dataStorage.gdprChildPmId
+
+        val usedPmId = childPmId ?: pmId
```

Comment:
Is this method used when the dev call `loadGDPRPrivacyManager`? If it is, we should always use the `pmId` the dev pass as parameter. 
The `childPmId` should only be used if the dev call the new method `loadGDPRGroupPrivacyManager` (same for CCPA).

Reply:
This is difficult, who need to have the priority? 
Should the user's pmId override the childPmId coming from the server or the opposite? (Right now I gave priority to the server)
What if the user keep using the pmId from the group and he gets the pm with the placeholder?